src/entities/utils/getPoolStateWithBalancesV2.ts

```diff
@@ -7,7 +7,8 @@ import {
 } from 'viem';
 import { PoolType } from '@/types';
 import { HumanAmount } from '@/data';
-import { CHAINS, VAULT } from '@/utils';
+import { CHAINS } from '@/utils';
+import { VAULT_V2 } from '@/utils/constantsV2';
```

Comment:
All these import changes could be avoided by applying the suggestion I made on the other comment -> export constantsV2 and constantsV3 files within utils/index.